### DISCUSSION TITLE
How do I enable watch-all-tests?

### DISCUSSION THREAD

**Discussion by ffMathy** [orta/vscode-jest#1063] [Q&A]
In the documentation, the mode `watch-all-tests` is mentioned as an alternative to just `watch-tests`. In other words, something that'll trigger `--watchAll` instead of `--watch`.

But how is this configured? The documentation doesn't state how.

**Comment by connectdotz** [comment]
You could customize the `jest.jestCommandLine` to append `--watchAll` there. Please also make sure your autoRun is not in watch mode, otherwise, it will override the `--watchAll` flag.

For example: in settings.json
```json
{
"jest.jestCommandLine": "yarn jest --watchAll",
"jest.autoRun": "on-save",
}
```
